### PR TITLE
feat: Scooter sheet redesign

### PR DESCRIPTION
**Pull request by KristianSelnas**
https://github.com/AtB-AS/kundevendt/issues/3545

### CODE REVIEW

File: src/vehicles/components/ScooterSheet.tsx [hunk 1]
```diff
@@ -144,27 +160,34 @@ const getRange = (rangeInMeters: number, language: Language) => {
...
 
 const useSheetStyle = StyleSheet.createThemeHook((theme) => ({
   button: {
     marginTop: theme.spacings.medium,
   },
-  container: {
+  vehicleInfo: {
     flexGrow: 1,
     flexShrink: 0,
+    gapCol: theme.spacings.medium,
```
Comment (rosvik) [inline src/vehicles/components/ScooterSheet.tsx:173]: Is this the property that doesn't work yet? If so, should it be removed for now, so it doesn't suddenly start to be behave in unexpected ways?

File: src/vehicles/components/ScooterSheet.tsx [hunk 2]
```diff
@@ -144,27 +160,34 @@ const getRange = (rangeInMeters: number, language: Language) => {
     rangeInMeters > 5000
       ? (rangeInMeters / 1000).toFixed(0)
       : formatDecimalNumber(rangeInMeters / 1000, language, 1);
-  return `ca ${rangeInKm}km`;
+  return `ca ${rangeInKm} km`;
```
Comment (rosvik) [inline src/vehicles/components/ScooterSheet.tsx:163]:
```suggestion
  return `ca. ${rangeInKm} km`;
```

**Review by rosvik — APPROVED** [review]
Looks good! Just a couple of nitpicks.

File: src/vehicles/components/ScooterSheet.tsx (hunk 1 above)
Comment (KristianSelnas) [reply src/vehicles/components/ScooterSheet.tsx:173]: You are right. This is just a left over from a previous iteration. I'll remove it.

**Pull request merged**